src/lib.rs

```diff
@@ -75,13 +75,6 @@ impl From<&NixFile> for PathBuf {
     }
 }
 
-/// Proxy through the `Display` class for `PathBuf`.
-impl std::fmt::Display for NixFile {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        PathBuf::from(self).display().fmt(f)
-    }
-}
```

Comment:
This is a symptom of adding another field to `NixFile`, even though it should be a separate enum.